python/tvm/relay/frontend/tensorflow.py

```diff
@@ -44,6 +44,28 @@
 __all__ = ["from_tensorflow"]
 
 
+def check_symbolic_shape(shape):
+    return not all([isinstance(dim, (int, tvm.tir.IntImm)) for dim in shape])
+
+
+def list_shape_of(tensor, ndim):
+    shape_tensor = _op.shape_of(tensor)
+    return [
+        _op.strided_slice(shape_tensor, begin=[i], end=[i + 1], strides=[1]) for i in range(ndim)
+    ]
```

Comment:
I'm not sure I understand this utility, but perhaps `take` would be more appropriate than `strided_slice`?